CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+cmake_minimum_required(VERSION 3.8)
+get_directory_property(is_subproject PARENT_DIRECTORY)
+
+project(frozen VERSION 1.0.0 LANGUAGES CXX)
+include(CTest)
+include(CMakePackageConfigHelpers)
+include(CMakeDependentOption)
+
+option(frozen.cpp17
+  "Use the C++ 17 standard or greater for projects linking to frozen")
```

Comment:
Should be C++14

Reply:
When this option is OFF, the C++ standard defaults to 14. I'll modify the doc string.